tests/ci/azure_pipeline_test/dsvm_notebook_linux_cpu.yml

```diff
@@ -6,52 +6,21 @@ pr:
 - main
 - staging
 
-#Any commit to this branch will trigger the build.
+# Any commit to this branch will trigger the build.
 trigger:
 - staging
 - main
 
 variables:
 - group: LinuxAgentPool
 
-jobs:
-- job: unit
-  displayName: 'Unit tests Linux CPU on notebooks'
-  timeoutInMinutes: 20 # how long to run the job before automatically cancelling
-  pool:
-    name: $(Agent_Pool)
-
-  steps:
-  - bash: |
-      echo "##vso[task.prependpath]/data/anaconda/bin"
-      conda env list
-    displayName: Add Conda to PATH
-  
-  # Uncomment if needed
-  # Conda creation can take around 10min
-  # - bash: |
-  #     python tools/generate_conda_file.py
-  #     conda env update -n reco_base -f reco_base.yaml
-  #   displayName: 'Creating Conda Environment with dependencies'
-
-  - script: |
-      . /anaconda/etc/profile.d/conda.sh && \
-      conda activate reco_base && \
-      pytest tests/unit --durations 0 -m "notebooks and not spark and not gpu" --junitxml=reports/test-unit.xml
-    displayName: 'Run Tests'
-
-  - task: PublishTestResults@2
-    displayName: 'Publish Test Results **/test-*.xml'
-    inputs:
-      testResultsFiles: '**/test-*.xml'
-      failTaskOnFailedTests: true
-    condition: succeededOrFailed()
-
-  # Uncomment if needed
-  # - script: |
-  #     conda env remove -n reco_cpu -y   
-  #   workingDirectory: tests
-  #   displayName: 'Conda remove'
-  #   continueOnError: true
-  #   condition: succeededOrFailed()
-  
\ No newline at end of file
+extends:
+  template: dsvm_linux_template.yml
+  parameters:
+    test_types:
+    - unit
+    task_name: "Test - Unit Notebook Linux CPU"
+    conda_env: "unit_notebook_linux_cpu"
+    conda_opts: "python=3.6"
```

Comment:
do you know if this will install 3.6.0 or it will install the latest 3.6.*?

Reply:
this installs the latest version of 3.6.* in anaconda channel (right now 3.6.13)